src/components/UnauthenticatedApp/UnauthenticatedApp.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { renderWithRouter } from 'testUtils';
+import '@testing-library/jest-dom/extend-expect';
+
+import UnauthenticatedApp from './UnauthenticatedApp';
+
+describe('Unauthenticated App', () => {
+  it('renders the Sign in Page', () => {
+    const { getByTestId } = renderWithRouter(<UnauthenticatedApp />, {
+      history: ['/sign-in'],
+    });
+    expect(getByTestId('signin-form')).toBeInTheDocument();
+  });
+  it('renders the Sign up Page', () => {
+    const { getByTestId } = renderWithRouter(<UnauthenticatedApp />, {
+      history: ['/sign-up'],
+    });
+    expect(getByTestId('signup-form')).toBeInTheDocument();
+  });
+  it('renders the Forgot Password Page', () => {
+    const { getByTestId } = renderWithRouter(<UnauthenticatedApp />, {
+      history: ['/forgot-password'],
+    });
+    expect(getByTestId('forgot-password-email-form')).toBeInTheDocument();
+  });
+  it('redirects to the Sign in Page when an unknown route is used', () => {
+    const { getByTestId } = renderWithRouter(<UnauthenticatedApp />, {
+      history: ['/no-match'],
+    });
+    expect(getByTestId('signin-form')).toBeInTheDocument();
+  });
```

Comment:
what about adding a test to check that an authenticated route redirects to the unauthenticated app as well?

Reply:
sorry, this isn't accurate... that test should be in App.test.js